modules/snooker/src/main/ts/ephox/snooker/util/CssUtils.ts

```diff
@@ -0,0 +1,21 @@
+import { HTMLElement, HTMLTableElement } from '@ephox/dom-globals';
+import { Arr } from '@ephox/katamari';
+import { SelectorFilter, Attr, Compare, Element } from '@ephox/sugar';
+
+const getPixelWidth = (elm: HTMLElement) => elm.getBoundingClientRect().width;
```

Comment:
These likely already exist elsewhere (eg `Sizes`), plus we don't normally use non-sugar elements in these libraries.